### ISSUE TITLE
2.1.12 Preview broken

### ISSUE THREAD

**Issue by buckett** [Studio-42/elFinder#1478]
In 2.1.11 preview is working correctly with a Java connector: https://github.com/bluejoe2008/elfinder-2.x-servlet/ however when I upgrade to 2.1.12 I don't see any preview icons and the preview functionality doesn't work. The unhelpful stack trace when clicking on preview for an image is:
````
RangeError: Maximum call stack size exceeded
    at RegExp.[Symbol.replace] (native)
    at RegExp.[Symbol.replace] (native)
    at String.replace (native)
    at Function.camelCase (http://code.jquery.com/jquery-1.8.0.min.js:2:14130)
    at Function.data (http://code.jquery.com/jquery-1.8.0.min.js:2:23622)
    at Function._data (http://code.jquery.com/jquery-1.8.0.min.js:2:24121)
    at Function.queue (http://code.jquery.com/jquery-1.8.0.min.js:2:25154)
    at Function.dequeue (http://code.jquery.com/jquery-1.8.0.min.js:2:25274)
    at HTMLDivElement.<anonymous> (http://code.jquery.com/jquery-1.8.0.min.js:2:88508)
    at Function.each (http://code.jquery.com/jquery-1.8.0.min.js:2:14473)
````

**Comment by buckett** [comment]
Ok, updating to newer versions of jquery/jquery-ui fixes the issue in 2.1.12 for me, the updates I made were:
jquery-ui 1.8.23 -> 1.11.4
jquery 1.8.0 -> 1.12.4

**Comment by nao-pon** [comment]
@buckett I got same error with jQuery/jQuery UI: 1.8.0/1.9.0. I'll fix it.

Thanks!
